packages/marko-web-algolia-site-search/graphql/fragments/content-list.js

```diff
@@ -0,0 +1,67 @@
+const gql = require('graphql-tag');
```

Comment:
Same for fragments -- these should be provided by the websites, as each site will likely have different field requirements depending on their layouts